### PR TITLE
Github actions CI added

### PR DESCRIPTION
**Pull request by kolosovpetro**
Github actions CI added

### CODE REVIEW

File: .github/workflows/dotnet-build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        with:
+          dotnet-version: '3.1.x'
+
+      - name: ♻ NuGet Cache ♻
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.NUGET_PACKAGES }}
+          key: ${{ runner.os }}.nuget.${{ hashFiles('**/*.csproj') }}
+
+      - name: 🔄 Restore Nuget Packages 🔄
```
Comment (maartenba) [inline .github/workflows/dotnet-build-and-test.yml]: We're using NUKE to build our project (https://www.nuke.build/index.html). Restore, build and test can be replaced with a call to `build.sh` and the `Test` target (https://github.com/JetBrains/space-dotnet-sdk/blob/main/build/Build.cs#L74)

**Review by maartenba — CHANGES_REQUESTED** [review]
Nice addition for folks wanting to contribute! Left a comment on changing to use NUKE so the build matches what we do on release.

File: .github/workflows/dotnet-build-and-test.yml (hunk 1 above)
Comment (kolosovpetro) [reply .github/workflows/dotnet-build-and-test.yml]: Hmm that's quite interesting, thanks for introducing Nuke to me :)

Comment (maartenba) [reply .github/workflows/dotnet-build-and-test.yml]: Thanks for the update!

**Pull request merged**